api/app/models/spree/api/webhooks/stock_item_decorator.rb

```diff
@@ -0,0 +1,29 @@
+module Spree
+  module Api
+    module Webhooks
+      module StockItemDecorator
+        def self.prepended(base)
+          base.around_save :queue_webhooks_requests_for_variant_backorderable!
+        end
+
+        private
+
+        def queue_webhooks_requests_for_variant_backorderable!
+          was_out_of_stock = !variant.in_stock_or_backorderable?
+          was_not_backorderable = !variant_backorderable?
+          yield
+          touch # changes must be reflected before instantiating the serializer
```

Comment:
@sebastian-palma this could create unexpected side effects (eg. executing some callbacks) or even create an infinite loop with this callback?

Reply:
maybe using a `.reload` will do

Reply:
That's right, `reload` does the trick, although by [SD-1544](https://spark-solutions.atlassian.net/browse/SD-1544) touch shouldn't emit webhooks if modifying only timestamps, isn't it?